test/evergreen.yml

```diff
@@ -4649,6 +4677,166 @@ tasks:
         vars:
           test_path: bench/wt2853_perf/wt2853_perf
 
+###################################################
+# Aggregate statistics for all performance tests  #
+###################################################
+
+  - name: aggregate-perf-stats
+    tags: ["aggregate-perf"]
+    depends_on:
+      - name: ".lsm-perf"
+      - name: ".btree-perf"
+      - name: ".oplog-perf"
+      - name: perf-test-update-checkpoint-btree
+      - name: perf-test-overflow-10k
+      - name: perf-test-overflow-130k
+      - name: perf-test-parallel-pop-stress
+      - name: perf-test-update-grow-stress 
+      - name: perf-test-update-shrink-stress
+      - name: perf-test-update-delta-mix1
+      - name: perf-test-update-delta-mix2
+      - name: perf-test-update-delta-mix3
+      - name: perf-test-multi-btree-zipfian
+      - name: perf-test-evict-btree-stress-multi
+      - name: perf-test-evict-btree
+      - name: perf-test-evict-btree-1
+      - name: ".log-perf"
+      - name: perf-test-long-500m-btree-populate
+      - name: perf-test-long-500m-btree-50r50u
+      - name: perf-test-long-500m-btree-50r50u-backup
+      - name: perf-test-long-500m-btree-80r20u
+      - name: perf-test-long-500m-btree-rdonly
+      - name: perf-test-long-checkpoint-stress
+      - name: many-dhandle-stress
+    commands:
+      - func: "get project"
```

Comment:
Do we not give the current_dir path, here?

Reply:
We don't need to, it will be operating in the home directory.